src/core/Akka.Remote.Tests/Properties/AssemblyInfo.cs

```diff
@@ -40,4 +42,4 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
-
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true)]
```

Comment:
This is a new XUnit2 thing, is it not? Is this just here primarily to avoid parallelization